monkey/agent_plugins/credentials_collectors/chrome/src/chrome_credentials_collector.py

```diff
@@ -0,0 +1,65 @@
+import time
+from typing import Sequence
+
+from common.agent_events import CredentialsStolenEvent
+from common.credentials import Credentials
+from common.event_queue import IAgentEventPublisher
+from common.tags import (
+    CREDENTIALS_FROM_PASSWORD_STORES_T1555_TAG,
+    DATA_FROM_LOCAL_SYSTEM_T1005_TAG,
+    UNSECURED_CREDENTIALS_T1552_TAG,
+)
+from common.types import AgentID, Event
+
+from .typedef import CredentialsDatabaseProcessorCallable, CredentialsDatabaseSelectorCallable
+
+CHROME_CREDETIALS_COLLECTOR_TAG = "chrome-credentials-collector"
+CHROME_COLLECTOR_EVENT_TAGS = frozenset(
+    (
+        CHROME_CREDETIALS_COLLECTOR_TAG,
+        DATA_FROM_LOCAL_SYSTEM_T1005_TAG,
+        UNSECURED_CREDENTIALS_T1552_TAG,
+        CREDENTIALS_FROM_PASSWORD_STORES_T1555_TAG,
+    )
+)
+
+
+class ChromeCredentialsCollector:
+    def __init__(
+        self,
+        select_credentials_database: CredentialsDatabaseSelectorCallable,
+        process_credentials_database: CredentialsDatabaseProcessorCallable,
+        agent_id: AgentID,
+        agent_event_publisher: IAgentEventPublisher,
```

Comment:
NIT: Put the agent id and event publisher first, and in the same order as plugin.py

Rational: Plugin.py builds this component by taking the set of parameters it's given and adding new ones. Preserving the order reduces cognitive load on the developer.